Plugins/NativeEngine/Source/IndexBuffer.cpp

```diff
@@ -32,9 +32,7 @@ namespace Babylon
                 bgfx::destroy(m_handle);
             }
         }
-
-        m_bytes.clear();
-
+        m_bytes = {};
```

Comment:
nit: `reset()`? We reset these elsewhere.
```suggestion
        m_bytes.reset();
```